docs/cita/configuration/service-config.md

```diff
@@ -18,6 +18,11 @@ $ ls 0
 
 ## Auth
 
```

Comment:
My suggestion is emphasizing the version, just note this is the latest config status
 
Because differences a lot between versions now. Users in earlier version may be confused about this config docs.

Reply:
That's ok. because this is the latest develop version. when we release v0.22.0, it will fetch the tag and it will only be exist on the v0.22.0 version.

Reply:
Every release version has it's own version doc.

Reply:
so to me , it no need change doc?

Reply:
let it go, No ned to change